lambdas/src/apis/collections/Utils.ts

```diff
@@ -78,7 +78,25 @@ export function createExternalContentUrl(
   return undefined
 }
 
-export function findHashForFile(entity: Entity, fileName: string | undefined) {
+export function createExternalLambdasUrl(urn: string): string {
+  const catalystUrl = baseLambdasServerUrl(process.env.CATALYST_URL) ?? 'http://localhost'
+  return `${catalystUrl}/lambdas/collections/contents/${urn}`
+}
+
+function baseLambdasServerUrl(url?: string): string | undefined {
+  if (!url) return
+
+  let configAddress = url.toLowerCase()
+  if (!configAddress.startsWith('http')) {
+    configAddress = 'http://' + configAddress
+  }
```

Comment:
why does this `if` exist? I think we would  benefit from break in case of  misconfiguration instead of trying to fix it

Reply:
Just wanted to be consistent with how we handle the cataylst url, same behaviour as the `SmartContentClient`. 
We can remove the logic tho.

Reply:
If the `configAddress` is read from the `.env` file, then I think that it's okay to support both cases. It's not a case of misconfiguration of the leading `/`, this is more like a sanitization.